README.md

```diff
@@ -32,792 +32,79 @@
-
-## Installing
-
-Good news everybody, we're on NPM.
-```
-npm install alsatian
-```
-
-## Use with JavaScript
-
-If you're using JavaScript, no worries you can still use Alsatian with Babel. Currently there is no official support for decorators (see [babel issue](https://github.com/babel/babel#2645)) but you can use a plugin!
-
-Add `transform-decorators-legacy` plugin
-
-```
-npm install babel-plugin-transform-decorators-legacy --save-dev
-```
-
-Then update your `.babelrc`
-
-```
-{
-  ...
-  "plugins": ["transform-decorators-legacy"]
-  ...
-}
-```
-
-You should now be able to use Alsatian decorators as in all the examples below, Hooray!
-
-## Running alsatian
-
-### CLI
-
-Alsatian has a CLI for easy use with your package.json or your favourite cli tool
-
-```
-alsatian [list of globs]
-
-alsatian "./test/**/*.spec.js" "./special-test.js"
-```
-
-#### CLI Options
-
-You can change how Alsatian runs your tests using the available options
-
-| Option                    | Alias  | Description                                                                  |
-| ------------------------- | ------ | ---------------------------------------------------------------------------- |
-| --help                    | -h     | Outputs info about how to use the CLI                                        |            
-| --version                 | -v     | Outputs the version of the CLI                                               |            
-| --tap                     | -T     | Will make Alsatian output in TAP format (to be consumed by a TAP reporter)   |             
-| --timeout [number in ms]  | -t     | Sets the maximum time that a test can run for before failing (default 500ms) |
-
-### Node.js
-
-If you're more of a nodey person then you can use that too
-
-```typescript
-import { TestSet, TestRunner } from "alsatian";
-import { TapBark } from "tap-bark";
-
-// create test set
-const testSet = TestSet.create();
+## Quick Start
 
-// add your tests
-testSet.addTestsFromFiles("./tests/you-want/to-add/**/*.spec.js");
+If you're loving TypeScript then pop on down to our [TypeScript Quick Start](https://github.com/alsatian-test/alsatian/wiki/typescript-setup).
 
-// create a test runner
-const testRunner = new TestRunner();
+Otherwise if you're more of a JavaScript kinda person, have a gander at our [JavaScript Quick Start](https://github.com/alsatian-test/alsatian/wiki/javascript-setup)
 
-// setup the output
-testRunner.outputStream
-          // this will use alsatian's default output if you remove this
-          // you'll get TAP or you can add your favourite TAP reporter in it's place
-          .pipe(TapBark.create().getPipeable()) 
-          // pipe to the console
-          .pipe(process.stdout);
+Full documentation can be found [here](https://github.com/alsatian-test/alsatian/wiki/) the examples are usually in TypeScript but should be pretty much the same code for JavaScript (just lose the access modifiers and types).
```

Comment:
Maybe change here to on our/the wiki. Reads nicer (in my opinion), and accessibility guidelines tend to say don't use here as link text